<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SizeOptions</class>
 <widget class="QDialog" name="SizeOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>236</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>312</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>312</width>
    <height>236</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Size Options</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>293</width>
     <height>217</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="AntennaSizeGroupBox">
          <property name="title">
           <string>Antenna Size Options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QRadioButton" name="GainMaxRadioButton">
                     <property name="text">
                      <string>Gain Max.</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                     <property name="autoRepeat">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="DiameterRadioButton">
                     <property name="text">
                      <string>Diameter</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="BeamWidthRadioButton">
                     <property name="text">
                      <string>Beamwidth</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <widget class="QLineEdit" name="GainLineEdit"/>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="DiameterLineEdit">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="BeamLineEdit">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QLabel" name="DbiGainLabel">
                     <property name="text">
                      <string>dBi</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="MeterDiameterLabel">
                     <property name="text">
                      <string>m</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="DegBeamLabel">
                     <property name="text">
                      <string>deg</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QDialogButtonBox" name="buttonBox">
             <property name="standardButtons">
              <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>GainMaxRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DiameterLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GainMaxRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>BeamLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GainMaxRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>GainLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DiameterRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>GainLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DiameterRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>BeamLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DiameterRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DiameterLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BeamWidthRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>BeamLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BeamWidthRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DiameterLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BeamWidthRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>GainLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SizeOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SizeOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
